src/include/distributed/multi_copy.h

```diff
@@ -43,8 +47,54 @@ typedef struct NodeAddress
 	int32 nodePort;
 } NodeAddress;
 
+/* CopyDestReceiver can be used to stream results into a distributed table */
+typedef struct CitusCopyDestReceiver
+{
+	DestReceiver pub;
+
+	/* relation and columns to which to copy */
```

Comment:
Added the DistTableCacheEntry, which removed the need for some of the other variables.